src/test/java/com/kaltura/client/test/tests/servicesTests/transactionHistoryTests/ListTests.java

```diff
@@ -0,0 +1,119 @@
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ListTests extends BaseTest{
+
+    private EntitlementFilter entitlementPpvFilter;
+    private TransactionHistoryFilter transactionHistoryFilter;
+    private Response<ListResponse<Entitlement>> listEntitlementServiceResponse;
+    private Response<ListResponse<BillingTransaction>> listBillingTransactionResponse;
+    private int numberOfDevicesInHousehold = 2;
+    private int numberOfUsersInHousehold = 2;
+
+    @BeforeClass
+    public void beforeClass(){
+        entitlementPpvFilter = new EntitlementFilter();
+        entitlementPpvFilter.setEntityReferenceEqual(EntityReferenceBy.HOUSEHOLD);  //Show entitlements per household
+        entitlementPpvFilter.setIsExpiredEqual(false);  //Don't show expired assets
+        transactionHistoryFilter = new TransactionHistoryFilter();
+        transactionHistoryFilter.setStartDateGreaterThanOrEqual((int)((System.currentTimeMillis()/1000)-86400));    //Start date from yesterday (86400 sec = 24 hours)
+        transactionHistoryFilter.setEndDateLessThanOrEqual((int)((System.currentTimeMillis()/1000)+86400)); //End date before tomorrow (86400 sec = 24 hours)
+    }
+
+    @Severity(SeverityLevel.BLOCKER)
+    @Test(description = "/transactionhistory/action/list - test ppv purchases is written correctly at transactionHistory")
+    public void purchase_ppv(){
+
+        Household household = HouseholdUtils.createHousehold(numberOfUsersInHousehold, numberOfDevicesInHousehold,true);
+        String masterUserKs = HouseholdUtils.getHouseholdMasterUserKs(household, HouseholdUtils.getDevicesListFromHouseHold(household).get(0).getUdid());   //Login with master to first device
+        String userKs = HouseholdUtils.getHouseholdUserKs(household, HouseholdUtils.getDevicesListFromHouseHold(household).get(1).getUdid());   //Login with user to second device
+
+        transactionHistoryFilter.setEntityReferenceEqual(EntityReferenceBy.HOUSEHOLD);  //All transactions per household
+        ListTransactionHistoryBuilder listTransactionHistoryBuilder = TransactionHistoryService.list(transactionHistoryFilter).setKs(masterUserKs);
+        listBillingTransactionResponse = executor.executeSync(listTransactionHistoryBuilder);   //transactionHistory/action/list for household
+        assertThat(listBillingTransactionResponse.error).isNull();
+        assertThat(listBillingTransactionResponse.results.getTotalCount()).isEqualTo(0);    //No purchases were performed
+
+        int assetFileId = getSharedWebMediaFile().getId();
+        PurchaseUtils.purchasePpv(userKs, Optional.of(getSharedMediaAsset().getId().intValue()), Optional.of(assetFileId), null);   //Purchase asset
+        String ppvModuleId = PurchaseUtils.purchasePpvDetailsMap.get("ppvModuleId");
```

Comment:
String constants - also make sense to use common (at least for that class) variables e.g.: 
public static final String PPV_MODULE_ID_KEY = "ppvModuleId";